recipes/moleculetda/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "moleculetda" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/moleculetda-{{ version }}.tar.gz
+  sha256: 43e2e2d21abbb1011421e98090013559c31658b350ce281ea874ec2a6c1dc1bb
+
+build:
+  entry_points:
+    - moleculetda = moleculetda.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win, py<36]
```

This looks to be pure Python and could be noarch:
```suggestion
  noarch: python
```

I'm not sure what caused any windows failures, but if it's due to missing dependencies, we can ignore those errors.